docs/overview.md

```diff
@@ -1402,33 +1358,6 @@ func baz(f foo) {
 
 
 `unexportedCall` is very opinionated.
-<a name="unlabelStmt-ref"></a>
```

Comment:
This removal shouldn't happen, please, update your master and regenerate docs one more 😉

Reply:
@cristaloleg I have branched off from the latest master and regenerated the docs again but it still removes this part